cmake/HeaderTest.cmake

```diff
@@ -1,5 +1,5 @@
 # #%L
-# OME Files C++ libraries (cmake build infrastructure)
+# Bio-Formats C++ libraries (cmake build infrastructure)
```

Comment:
No, it's from copying it from another component.  I've switched it to the more generic "OME" so we can share these between components without naming issues.